src/refract/api.es6

```diff
@@ -84,13 +89,27 @@ class HttpMessagePayload extends ApiBaseArray {
   }
 
   header(name) {
-    const header = this.attributes.headers.content.filter(filterBy.bind(this, {
-      name,
-      ignoreCase: true
-    }))[0];
+    const headers = this.attributes.headers;
+    let header = null;
+
+    if (headers) {
+      header = headers.content.filter(filterBy.bind(this, {
+        name,
+        ignoreCase: true
+      }))[0];
+    }
+
     return header ? header.content : header;
   }
 
+  get contentType() {
+    if (this.header('Content-Type')) {
```

Comment:
This is using `this.header(name)`, which in turn ignores casing when filtering the header names, so it's safe :wink: